app/editor/src/features/admin/filters/utils/generateQuery.ts

```diff
@@ -0,0 +1,151 @@
+import moment from 'moment';
+import { IFilterSettingsModel } from 'tno-core';
+
+/**
+ * Generates an Elasticsearch query based on specified 'query'.
+ * @param settings Form values that will be used to configure the elasticsearch query.
+ * @param query Original query object.
+ * @returns Elasticsearch query JSON.
+ */
+export const generateQuery = (settings: IFilterSettingsModel, query: any = {}) => {
```

Comment:
@jdtoombs This is an example of generate an Elasticsearch query from an advanced search form.  We should place this type of code in our shared library so that we can maintain it in a single place.